test/e2e-v2/java-test-service/Dockerfile.consumer

```diff
@@ -0,0 +1,26 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG SW_AGENT_JDK_VERSION=8
+
+FROM ghcr.io/apache/skywalking-java/jdk-${SW_AGENT_JDK_VERSION}:f19195368916e9514d19084eb1a29737c74bd25b
```

Comment:
Let's move `f19195368916e9514d19084eb1a29737c74bd25b` to a system environment, controlling by GHA file. And we need to update commit tag to `3997f0256056788bd054ee37e4603c11c0fd6756`, which mean, Tomcat/SpringMVC's operation name should add HTTP method as prefix, see https://github.com/apache/skywalking-java/pull/37

Reply:
Change to using system environment to management, please check: https://github.com/apache/skywalking/pull/7810/files#diff-29a4ad0b26179d51ca486039a03d45f783f02a5906e198292001358a29673c2fR40-R45
E2E cases have been fixed.